support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -105,11 +105,17 @@ export type RegularPaymentRequest = {|
 export type StripePaymentMethod = 'StripeCheckout' | 'StripeApplePay' | 'StripePaymentRequestButton';
 export type StripePaymentRequestButtonMethod = 'none' | StripePaymentMethod;
 
-export type StripeAuthorisation = {|
```

Comment:
Split `StripeAuthorisation` into `StripeCheckoutAuthorisation` and `StripePaymentIntentAuthorisation` because one needs a `token` and one needs a `paymentMethodId`. After migration, `StripeCheckoutAuthorisation` can be removed